config/routes.rb

```diff
@@ -2,7 +2,9 @@
   root "mountains#index"
   devise_for :users, :controllers => { :registrations => :registrations}
 
-  resources :mountains
+  resources :mountains do
```

Comment:
Youll want to also edit these routes, such that they dont expose paths in your app that go nowhere